person.rb

```diff
@@ -0,0 +1,34 @@
+require './nameable'
+
+class Person < Nameable
+  attr_reader :id
+  attr_accessor :name, :age, :rentals
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super()
+    @id = Random.rand(1...1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
+    rental.person = self
+  end
```

Comment:
- [ ] Let's add the logic of creating a new rental in this method which means that this method should receive two arguments (book, date) so that we can create a new rental. See hint on how you might structure this method.

```ruby
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

Reply:
Thanks for the feedback